src/stories/Icon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { IconProps } from "./interfaces";
+
+const Icon = (IconProps: IconProps) => {
+  const { iconLink } = IconProps;
+
+  return <img src={iconLink} alt="icon" />;
```

Comment:
I think it would be better to insert icons as inline SVG. It allows you to dynamically change icon colour (e.g. on hover). For this purpose you can import SVG as a React component like so:
```ts
import { ReactComponent as MyIcon } from "./icon.svg"
```

Reply:
In this case MyIcon is react component that is ready for use on the level where we will add it. It's correct?

If it's so, we can get rid of Icon component and use just this approach `import { ReactComponent as MyIcon } from "./icon.svg"`

Reply:
@antkl please, take a look at the above comment :)